actionview/lib/action_view/helpers/number_helper.rb

```diff
@@ -117,6 +120,8 @@ def number_to_phone(number, options = {})
       #   # => R$1234567890,50
       #   number_to_currency(1234567890.50, unit: "R$", separator: ",", delimiter: "", format: "%n %u")
       #   # => 1234567890,50 R$
+      #   number_to_currency(1234567890.50 ,strip_insignificant_zeros: true)
```

Comment:
Can you fix the spacing around the comma in both places you add this line?

> number_to_currency(1234567890.50 ,strip_insignificant_zeros: true)

Should be

> number_to_currency(1234567890.50, strip_insignificant_zeros: true)

Reply:
Updated!